_posts/2017/2017-11-12-use-a-pull-request-template.markdown

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Use a pull request template'
+description: 'It helps you answer what, why & how.'
+layout: article
+category: ruby
+image:
+  base: '2017/use-a-pull-request-template'
+  alt: 'Man pulling rope'
+  credit: 'Stijn Swinnen'
+  source: 'https://unsplash.com/photos/Q8FHN3qSq2w'
+---
+
+Pull Requests are the primary place where you explain your work. The audience for a Pull Request can be your immediate co-workers, other interested parties or even your future self.
```

Comment:
I think you need to edit 'explain your work' to something else. Maybe be more explicit and get across that you're talking about when you're submitting work for review, etc. Because, if you're just pushing into master, there's no pull request...

Reply:
Take out the capitals on 'Pull Request' here.